src/create-app.ts

```diff
@@ -112,3 +123,44 @@ async function prepare(starter: Starter) {
 function validateProjectName(projectName: string) {
   return !/[^a-zA-Z0-9-]/.test(projectName);
 }
+
+/**
+ * Helper for performing for running through the necessary steps to create a git repository for a new project
+ * @param directory the name of the new project's directory
+ * @returns true if no issues were encountered, false otherwise
+ */
+const initGitForStarter = (directory: string): boolean => {
```

Comment:
The boolean return values for each of these helpers isn't my favorite. Without adding a more complex return type from each of them (like Alice's `Result` type we added to Stencil core), we expect the util fn to capture the error/log it, and return a sentinel value. I went back and forth on adding something more complex, and decided to keep it simple for now

Reply:
I had, but I felt like I was starting to complicate the code more than I needed to. Happy to change it if you feel strongly about it, but I think this could be part of the exercise to moving to something like `simple-git` in the future